src/neo/VM/Helper.cs

```diff
@@ -280,13 +280,14 @@ private static ContractParameter ToParameter(StackItem item, List<(StackItem, Co
                     };
                     break;
                 case Null _:
+                case null:
```

Comment:
How can `item` be `null`? I think the caller should not send `null` as a parameter to this method.

Reply:
This can be null in the old Notify arguments, shall we throw an exception for this?

Reply:
I think it shouldn't be `null`. It should throw an exception.